### ISSUE TITLE
paths vs PATHS

### ISSUE THREAD

**Issue by MatthijsBurgh** [rec/impall#23]
In 26e5368dab9af6ca62b50a6fa786cb438ad406f4 you removed the `.upper()` from the attr. As you defined paths, lower case, 
https://github.com/rec/impall/blob/26e5368dab9af6ca62b50a6fa786cb438ad406f4/impall.py#L339
as the argparse argument, this isn't mapped to the PATHS attribute in the `ImpAllTest` class.

Therefore the paths argument is broken. You should reintroduce the `.upper()` or change the argparse argument to `PATHS`.

Currently only `-P` or `--PATHS` works.

**Comment by rec** [comment]
Hello, again, and ARG, what an ineptness on my part, due simply to overload.

On it!

**Comment by rec** [comment]
I should add that this is one of my older libraries, and I was also determined to have no dependencies, so I used argparse, which I now consider somewhat bad, and then did all these tricks.

Downright embarrassing, oh, well.

**Comment by rec** [comment]
I fixed this and tested it as follows (see #7)

```
impall .
impall -p .
impall -P .
impall --PATH .
impall test
impall test/test_impall.py
```
Note that the last one does nothing, see #11.

This is released as 1.5.1.

I am hoping to have some time in the new year to be able to fix those two issues above, but please contact me if anything is blocking you immediately and I'll fast-track it.

**Comment by MatthijsBurgh** [comment]
Thanks!
